dishdive/app/restaurants/[business_id]/page.tsx

```diff
@@ -0,0 +1,197 @@
+"use client";
+
+import MapComponent from "@/components/map";
+import Rating from "@/components/rating";
+import { Badge } from "@/components/ui/badge";
+import { useChatContext } from "@/context/chatContext";
+import { SlLocationPin } from "react-icons/sl";
+
+interface RestaurantPageProps {
+  params: {
+    business_id: string;
```

Comment:
Use camelCasing in JS
```suggestion
    businessId: string;
```